content/posts/why-meson/index.md

```diff
@@ -0,0 +1,248 @@
++++
+"author" = "ret2libc"
+"title" = "Why Meson"
+"date" = "2021-01-23"
+"summary" = "Why we switched to Meson/Ninja as our main build system"
+"tags" = ["dev", "rizin"]
+
+"ShowToc" = false
+"weight" = 2
++++
+
+As developers, we think it is essential to have a building system that eases
+our work, allows us to compile Rizin quickly on a wide range of devices, is
+easy to understand and to modify, and provides a nice set of features one
+would usually expect from a full-fledged building system. Since its
+inception, Rizin has focused on improving its Meson build files and making
+its support first-class, while deprecating the original building system used
+in radare2. In the following article we will explain the reasons behind this
+choice and the key benefits of Meson.
+
+# TL;DR
+
+- Meson is declarative and easy to understand
+- Meson is fast, no files are recompiled if not necessary
```

Comment:
Isn't it `ninja` which is responsible of checking if it should recompile a file or not?